app/models/user.rb

```diff
@@ -35,6 +35,9 @@ class User < ActiveRecord::Base
          :password_archivable, # in signonotron2/lib/devise/models/password_archivable.rb
          :password_expirable   # in signonotron2/lib/devise/models/password_expirable.rb
 
+  validates :require_2sv,
```

Comment:
I don't think this validation is correct any more. Admins should be able to flag an already-enrolled user as requiring 2SV